deepcell/applications/application_test.py

```diff
@@ -243,13 +244,43 @@ def _format_model_output(Lx):
         y = app._format_model_output(x)
         self.assertAllEqual(x, y['inner-distance'])
 
+    def test_batch_predict(self):
+
+        def predict1(x, batch_size=4):
+            y = np.random.rand(*x.shape)
+            return [y]
+
+        def predict2(x, batch_size=4):
+            y = np.random.rand(*x.shape)
+            return [y] * 2
+
+        for num_images in [4, 8, 10]:
+            for num_pred_heads in [1, 2]:
+                model = DummyModel(n_out=num_pred_heads)
+                app = Application(model)
+
+                x = np.random.rand(num_images, 128, 128, 1)
+
+                if num_pred_heads == 1:
+                    app.model.predict = Mock(side_effect=predict1)
+                else:
+                    app.model.predict = Mock(side_effect=predict2)
+                y = app._batch_predict(x, batch_size=4)
+                self.assertEqual(x.shape, y[0].shape)
+
+                assert app.model.predict.call_count == np.ceil(num_images / 4)
```

Comment:
Please set batch_size equal to a variable and include in the `np.ceil(num_images / batch_size)`?